device/ufispace/x86_64-ufispace_s9180_32x-r0/pmon_daemon_control.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
The default value for most of the fields is 'false'. You dont need to mention each one separately. Only this would have sufficed. 

{
    "skip_ledd": "true"
}